src/components/About/About.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from './About.module.css';
+
+const About = () => (
+    <div className={styles.wrapper}>
+        <h1>Обо мне</h1>
+    </div>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)